### PR TITLE
title attributes for keyboard shortcuts

### PR DESCRIPTION
**Pull request by kasperpeulen**
Maybe not a final design, but maybe nice to give the users some kind of way to find out what the shortcuts are (without having to read the code ! :P )

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -69,7 +69,7 @@
           <div flex></div>
           <div id="dartbusy" class="busylight"></div>
           <div>
-            <button type="button" id="runbutton">
+            <button type="button" id="runbutton" title="CTRL+ENTER / ⌘+ENTER">
```
Comment (devoncarew) [inline web/index.html]: How about `Run (ctrl-enter / ⌘-enter)`?

File: web/index.html [hunk 2]
```diff
@@ -95,8 +95,8 @@
         </div>
         <div class="view" flex two layout vertical>
           <div class="header" layout horizontal wrap>
-            <li><a id="consoletab" selected><span>Console</span></a></li>
-            <li><a id="doctab"><span>Documentation</span></a></li>
+            <li><a id="consoletab" title="CTRL+ENTER / ⌘+ENTER" selected><span>Console</span></a></li>
```
Comment (devoncarew) [inline web/index.html]: I think we're ok with no title on this tab - it switches to this tab as a side-effect of running the app.

File: web/index.html [hunk 3]
```diff
@@ -95,8 +95,8 @@
...
+            <li><a id="doctab" title="F1"><span>Documentation</span></a></li>
```
Comment (devoncarew) [inline web/index.html]: `Documentation (F1)`?

**Comment by devoncarew** [comment]
Good idea! I'd like to tweak the text a bit.

At some point we may have a small keyboard icon on the bottom (on the status line). That would open a modal overlay with the available keyboard shortcuts. We have very few right now, so it's not a critical feature. And, having the tab for documentation make that pretty discoverable.

**Comment by kasperpeulen** [comment]
@devoncarew Cool, changed it

**Comment by devoncarew** [comment]
wesome - thanks!

**Pull request merged**